.cspell.json

```diff
@@ -91,6 +91,7 @@
         "CISETUP",
         "isclass",
         "DSCs",
-        "levelno"
+        "levelno",
+        "uefibuilder"
```

Comment:
since your function is GetUefiBulderClass which is handled by spelling because of capitalization i would prefer that you use UefiBuilder in your comment and therefore this isn't needed.

Reply:
Will remove